tests/countries/test_bermuda.py

```
@@ -237,3 +237,32 @@ def test_2025(self):
+            ("2024-03-29", "Good Friday"),
+            ("2024-05-24", "Bermuda Day"),
+            ("2024-06-17", "National Heroes Day"),
+            ("2024-08-01", "Emancipation Day"),
+            ("2024-08-02", "Mary Prince Day"),
+            ("2024-09-02", "Labour Day"),
+            ("2024-11-11", "Remembrance Day"),
+            ("2024-12-25", "Christmas Day"),
+            ("2024-12-26", "Boxing Day"),
+        )
+
+    def test_l10n_en_us(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)